pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1484,6 +1484,7 @@ type CleanupPolicySpec struct {
 	// AllowUninstallWithVolumes defines whether we can proceed with the uninstall if they are RBD images still present
 	// +optional
 	AllowUninstallWithVolumes bool `json:"allowUninstallWithVolumes,omitempty"`
+	// TODO: add uninstall option AllowUninstallWithDependentResources
```

Comment:
We could use the existing `AllowUninstallWithVolumes` option, but IMO, that is a slightly different feature as the var is named.

In hindsight, it may have been better to have called that `ForceDelete`, `AllowUnsafeUninstall`, or `AllowUninstallWithUserData` to allow it to apply to future considerations like this one.

Having said that, maybe it would be best to call the new option `ForceDelete` or `AllowUninstallWithUserData`. This option should also imply `AllowUninstallWithVolumes`, and we might deprecate that option, though I wonder if we should keep it for backwards compatibility even if deprecated.